.github/workflows/vercel-fail-comment.yml

```
@@ -0,0 +1,394 @@
+                  .[] |
+                  select(.type == "stdout" or .type == "stderr") |
+                  .payload.text // .text // empty
+                ' /tmp/events.json 2>/dev/null | tail -50 || echo "")
+              fi
+              
+              # Limit output to last 100 lines to avoid huge comments
+              if [ -n "$error_logs" ]; then
+                error_logs=$(echo "$error_logs" | tail -100)
+              else
+                error_logs="No detailed error messages found in deployment logs.
+Please check the Vercel dashboard for complete build output."
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)